__tests__/Unit/Components/Tasks/TaskDetails.test.tsx

```diff
@@ -150,17 +159,6 @@ describe('TaskDetails Page', () => {
         });
     });
 
-    it('Renders N/A when link is empty or undefined', async () => {
-        const { getByText } = renderWithRouter(
-            <Provider store={store()}>
-                <Details detailType={'Link'} value={undefined} />
-            </Provider>
-        );
-        await waitFor(() => {
-            expect(getByText('N/A')).toBeInTheDocument();
-        });
-    });
-
```

Comment:
why you remove this test?

Reply:
Please check tests code, I have re-wrote all of these tests

Reply:
you can modify existing tests right?

Reply:
@sahsisunny yes, I have modified the existing tests according to my implementations